resource-managers/kubernetes/core/pom.xml

```diff
@@ -29,7 +29,7 @@
   <name>Spark Project Kubernetes</name>
   <properties>
     <sbt.project.name>kubernetes</sbt.project.name>
-    <kubernetes.client.version>1.4.34</kubernetes.client.version>
+    <kubernetes.client.version>2.0.3</kubernetes.client.version>
```

Comment:
Looks like some packages moved around related to Jobs, but we don't use those so it didn't affect us.

https://github.com/fabric8io/kubernetes-client/pull/646